backend/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -21,15 +21,10 @@
       "description": "Definitions of graphql scalar types implemented by graphql-datetime"
     },
     {
-      "name": "simple-report.site-admin-emails",
+      "name": "simple-report.demo-site-admin-emails",
```

Comment:
This block should just be deleted—it's now part of the DemoUserConfiguration object, which is covered on line 34/29 below.